js/modules/k6/browser/common/execution_context.go

```diff
@@ -113,6 +113,12 @@ func (e *ExecutionContext) adoptBackendNodeID(backendNodeID cdp.BackendNodeID) (
 		return nil, fmt.Errorf("resolving DOM node: %w", err)
 	}
 
+	// This can occur due to race conditions between trying to click on an element
+	// and chrome moving on (e.g. navigating).
+	if remoteObj == nil {
+		return nil, fmt.Errorf("the page may have navigated away or the element is now missing: %w", err)
```

Comment:
Are there any useful details we could let users know to help them prevent such errors, or at least give a better picture of why that error happens?

Reply:
There can be multiple reasons for this occurring, so it's not easy to add into an error -- load generator overloaded; chrome overloaded; use of elementHandle instead of locators.

I think for now we can return a new error, and wait for feedback to help us understand where exactly users are hitting this issue. At the moment I've seen this occur when chrome is overloaded, but the load generator is fine.

Reply:
If we don't add this information, then we might not fully respect what the description in What/Why is reporting.

What about the following form?

```suggestion
		return nil, fmt.Errorf("the page may have navigated away or the element is now missing. It might happen when k6 and/or Chrome are overloaded. You might need to increase the compute resources: %w", err)
```